feature/profile/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    alias(libs.plugins.mifospay.android.feature)
+    alias(libs.plugins.mifospay.android.library.compose)
+}
+
+android {
+    namespace = "org.mifospay.profile"
+}
+
+dependencies {
+    implementation(projects.core.data)
+    implementation(libs.squareup.okhttp)
+    implementation(libs.compose.country.code.picker)
+    implementation(libs.compose.material)
+    implementation(libs.coil.kt.compose)
+    implementation(libs.androidx.appcompat)
```

Comment:
why ?

Reply:
We need these (despite profile being child module) : -
1) implementation(libs.compose.country.code.picker) -> for togi countrypicker implementation
2)implementation(libs.coil.kt.compose) -> AsyncImage implementation
3)implementation(libs.compose.material) -> can't use material  3 in togi conuntrypicker for ' TextFieldDefaults'